app/pb/communities.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+service Communities {
+  // This is a secure service: a user needs to be authenticated and not jailed to call functions here, refer to auth.proto and jail.proto
+
+  // Discussion threads are organized as a tree with maximum three levels depth:
+  // thread1
+  // +- comment1
+  // |  +- reply1
+  // |  +- reply2
+  // +- comment2
+  //    +- reply3
+  //    +- reply4
+  // thread2
+  // +- comment3
+  // |  +- reply5
+  // |  +- reply6
+  // +- comment4
+  //    +- reply7
+  //    +- reply8
+
+  // The same API is used regardless of level. All posts are using the same id namespace.
+
+  // Get non-deleted posts belonging to a thread, sorted in reverse time order.
+  // Pagination is supported by the posted_before_thread_id and number fields.
+  rpc ReadThread(ReadThreadReq) returns (ReadThreadRes);
```

Comment:
Why `ReadThread` and not `GetThread`? I think `Get*` is more consistent and better due to its ubiquity in HTTP, etc.